appveyor.yml

```diff
@@ -0,0 +1,34 @@
+branches:
+  only:
+    - master
+
+environment:
+  ZEUS_HOOK_BASE:
+    secure: xxx
+  matrix:
+    - nodejs_version: "10"
+    - nodejs_version: "8"
+
+build:
+  verbosity: minimal
+
+install:
+  - ps: Install-Product node $env:nodejs_version
+  - npm install -g yarn
+  - node --version
+  - npm --version
+  - yarn --version
+
+build_script:
+  - yarn install
+  - yarn start job update --status=pending -B "%APPVEYOR_PULL_REQUEST_TITLE%" -J "%APPVEYOR_JOB_NAME%"
```

Comment:
Love how we're dogfooding this here. However that also means that if there is a bug with the current implementation, we end up with inconsistent jobs in Zeus.